crates/apollo-router/src/graph_factory.rs

```diff
@@ -30,6 +37,7 @@ impl GraphFactory<graphql::FederatedGraph> for FederatedGraphFactory {
         schema: Arc<graphql::Schema>,
     ) -> graphql::FederatedGraph {
         let service_registry = HttpServiceRegistry::new(configuration);
+        /*
         tokio::task::spawn_blocking(|| {
```

Comment:
Why remove the spawn_blocking here? It is necessary, the cal to the query planner is blocking right now, it creates and calls into a v8 vm